runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/PrimitiveParDoSingleFactory.java

```diff
@@ -177,10 +177,7 @@ public String getUrn(ParDoSingle<?, ?> transform) {
       // TODO: Is there a better way to do this?
       Set<String> allInputs =
           transform.getInputs().keySet().stream().map(TupleTag::getId).collect(Collectors.toSet());
-      Set<String> sideInputs =
-          parDo.getSideInputs().stream()
-              .map(s -> s.getTagInternal().getId())
-              .collect(Collectors.toSet());
+      Set<String> sideInputs = parDo.getSideInputs().keySet();
```

Comment:
I think this is wrong - we should still get this Set from getTagInternal like before.